src/containers/WelcomePage/components/LastUsedContent.tsx

```diff
@@ -20,9 +22,15 @@ import { classes } from '../WelcomePage';
 interface Props {
   articleId: number;
   locale: string;
+  userAccess: string;
 }
 
-const LastUsedContent: FC<Props & tType> = ({ articleId, locale, t }) => {
+const LastUsedContent: FC<Props & tType> = ({
+  articleId,
+  locale,
+  userAccess,
+  t,
```

Comment:
Propen t kjem fra injectT og er ikkje definert som en prop i Props. Trudde du måtte ha med injectT for at dette skulle funke.

Ellers forsøker vi å skrive oss bort fra redux og unngå mapStateToProps og heller bruke React Context. Det er greit her, for eg har innført en UserAccessContect i en anna pr, så koden her kan byttes ut med det når den er merga inn.

Reply:
Ref denne commiten: https://github.com/NDLANO/editorial-frontend/pull/813/commits/1bb68a29208ceffc044799d05ad160f1ec3aecdc

Reply:
Det så ut som om context kunne vært relevant for t og userAccess ja. Men er vel ikke alt det er like relevant for

Reply:
For å forklare endringen på t, så henter jeg den fra parent i stedet for å forenkle redux mappinga

Reply:
Eg skjønte det, men eg meinte at å sende t som prop til andre komponenter (nesten) ikkje gjøres i ed, og er heller ikkje meininga.